scripts/check_menhir_conflicts.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Expected input: first arg is the file with menhir error messages
+#                 second arg is the max allowed conflicts
+#                 third arg is the language
+FILE=$1
+num=$2
+lang=$3
+generated_ml_file=$4
+generated_mli_file=$5
+INSTR="To bypass this, manually set NUM_PERMITTED_CONFLICTS in pfff/lang_$lang/parsing/Makefile"
+
+# Parse conflicts output
+# Expect "Warning: [num] states have shift/reduce conflicts"
+# and/or "Warning: [num] states have reduce/reduce conflicts"
```

Comment:
Actually I'm more interested in this line:
Warning: 2 shift/reduce conflicts were arbitrarily resolved.

This is because sometimes you can have 2 states with shift/reduce conflicts, but both of those states
have each 10 shift/reduce conflicts, so the total is actuall 20 shift/reduce conflicts, not 2.

Reply:
Oh, I thought that it was saying "2 states have shift/reduce conflicts; 2 shift/reduce conflicts were arbitrarily resolved" (the same 2 states). Should I have it also sum the conflicts from "Warning: 2 shift/reduce conflicts were arbitrarily resolved"?

Reply:
Actually that's the only line that matters. I don't really care about the number of states with conflicts, I just care about the total number of actual shift/reduce (or reduce/reduce) conflicts, that were arbitrarily resolved.